daemon/rpc_set_postquantum_vpn.go

```diff
@@ -0,0 +1,45 @@
+package daemon
+
+import (
+	"context"
+	"log"
+
+	"github.com/NordSecurity/nordvpn-linux/config"
+	"github.com/NordSecurity/nordvpn-linux/daemon/pb"
+	"github.com/NordSecurity/nordvpn-linux/internal"
+)
+
+func (r *RPC) SetPostQuantumVpn(ctx context.Context, in *pb.SetGenericRequest) (*pb.Payload, error) {
+	var cfg config.Config
+	if err := r.cm.Load(&cfg); err != nil {
+		log.Println(internal.ErrorPrefix, err)
+	}
+
+	if cfg.AutoConnectData.PostquantumVpn == in.GetEnabled() {
+		return &pb.Payload{Type: internal.CodeNothingToDo}, nil
+	}
+
+	if cfg.Mesh && in.GetEnabled() {
+		return &pb.Payload{Type: internal.CodeConflict}, nil
+	}
+
+	if cfg.AutoConnect && in.GetEnabled() {
+		//TODO(msz): validate that server supports PQ
```

Comment:
ToDo?

Reply:
Yeah, I don't know what to do with this, since currently there's no server running with PQ (except maybe one?) support and the thing is that with PQ enabled we cannot connect to non-PQ servers.
I honestly don't know what to do with this fact and I wanted to discuss it.

Reply:
So it seems that all servers will support this feature. No need for the check. I'll remove it

Reply:
Removed